test/runtests.jl

```diff
@@ -13,6 +13,8 @@ struct TwoThirds a; b; c; end
 Functors.@functor TwoThirds (a, c)
 Optimisers.trainable(x::TwoThirds) = (a = x.a,)
 
+Yota_gradient(f, xs...) = Base.tail(Yota.grad(f, xs...)[2])
```

Comment:
Maybe this is a better rough translation function:
```suggestion
Yota_gradient(f, xs...) = map(y2z, Base.tail(Yota.grad(f, xs...)[2]));
y2z(::AbstractZero) = nothing  # we don't care about different flavours
y2z(t::Tangent) = map(y2z, ChainRulesCore.backing(canonicalize(t)))
y2z(x) = x
```
The only goal is to have as few changes as possible between tests using Zygote and the same with Yota.